lib/portlayer/exec/container_cache.go

```diff
@@ -48,11 +61,35 @@ func NewContainerCache() {
 func (conCache *containerCache) Container(idOrRef string) *Container {
 	conCache.m.RLock()
 	defer conCache.m.RUnlock()
-	// find by id
+	// find by id or moref
 	container := conCache.cache[idOrRef]
 	return container
 }
 
+// return an array of pointers to the containers in the cache
+// TODO: implement filtering here?
+func (conCache *containerCache) Containers(allStates *bool) []*Container {
```

Comment:
ok - so I THOUGHT because it was the package name that's why the other areas used sess instead of session, but apparently that's not the case.  A package alias wan't required, so I've changed it to session